project/apps/web/app/question/[id]/page.tsx

```diff
@@ -51,10 +52,12 @@ const QuestionPageContent = ({ id }: { id: string }) => {
         queryKey: [QUERY_KEYS.Question, id],
       });
       setEditModalOpen(false);
+      toast({ variant: "success", title: "Success", description: "Question updated successfully" });
```

Comment:
i think this should be doing `const { toast } = useToast() ` instead so that we are calling it within a react hook. if im not wrong the other toast if u want to use it outside of react components.

Reply:
ill make the change